src/gpu-compute/register_file.cc

```diff
@@ -194,6 +194,10 @@ RegisterFile::RegisterFileStats::RegisterFileStats(statistics::Group *parent)
     : statistics::Group(parent),
       ADD_STAT(registerReads,
               "Total number of DWORDs read from register file"),
+      ADD_STAT(cache_read_hits,
```

Comment:
To avoid confusion, I recommend calling these "rfc_..." instead, otherwise folks will likely think cache refers to the L1/L2/etc.